lib/StrtoArray.js

```diff
@@ -0,0 +1,16 @@
+function StrtoArray(string){
```

Comment:
In JavaScript, functions are written using `camelCase`, so the convention would be to call this `strToArray`.